src/tmx/TmxUtils/src/rsu/NTCIP_1218_MIB.h

```diff
@@ -1,45 +1,66 @@
 #pragma once
+
+#include <string>
+
 namespace tmx::utils::rsu::mib::ntcip1218
 {
     /**
      * @brief This header file contains a subset of RSU-specific NTCIP 1218 MIB OIDs from https://www.ntcip.org/file/2021/01/NTCIP-1218v0138-RSU-toUSDOT-20200905.pdf
      */
 
     // RSU Identifier. Contains the ID given to this RSU.
-    static constexpr const char *rsuIDOid = "1.3.6.1.4.1.1206.4.2.18.13.4.0";
+    static const std::string rsuIDOid = "1.3.6.1.4.1.1206.4.2.18.13.4.0";
```

Comment:
Why not prefer to use compile time evaluation with constexpr?

Reply:
Compile time evaluation is not required for these OIDs.  After googling this it seems these are just general constants and therefore do not need compile time evaluation. Please let me know if there is something here that I am missing.
![image](https://github.com/user-attachments/assets/ab937f12-c82c-4e5f-8f34-9a2b5566e17e)